Fix Handling Quit Requests wording about mobile platforms (#8980)
godotengine/godot-docs: tutorials/inputs/handling_quit_requests.rst

Most platforms have the option to request the application to quit. On
desktops, this is usually done with the "x" icon on the window title bar.

> On mobile devices, the app can be quit at any time while it is suspended

Reviewer: ```suggestion
On mobile devices, the app can quit at any time while it is suspended
```

    GetTree().AutoAcceptQuit = false;

> On Mobile Devices

Reviewer: ```suggestion
On mobile devices
```

-----------------

> There is no direct equivalent of ``NOTIFICATION_WM_CLOSE_REQUEST`` on mobile 

Reviewer: ```suggestion
There is no direct equivalent to ``NOTIFICATION_WM_CLOSE_REQUEST`` on mobile 
```

platforms. Due to the nature of mobile operating systems, the only place 
that you can run code prior to quitting is when the app is being suspended to 
the background. On both Android and iOS, the app can be killed while suspended 
at any time by either the user or the OS. A way to plan ahead for this 
possibility is to utilize ``NOTIFICATION_APPLICATION_PAUSED`` in order to 

> perform any needed code as the app is suspended.

Reviewer: ```suggestion
perform any needed actions as the app is being suspended.
```

On Android, pressing the Back button will exit the application if 

> Application > Config > Quit On Go Back is checked in the Project Settings 

Reviewer: ```suggestion
**Application > Config > Quit** On Go Back is checked in the Project Settings 
```
Don't remove this
